todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/taskdetail/TaskDetailFragment.java

```diff
@@ -68,13 +89,60 @@ public static TaskDetailFragment newInstance(String taskId) {
     @Override
     public void onResume() {
         super.onResume();
-        mPresenter.subscribe();
+        bind();
     }
 
     @Override
     public void onPause() {
+        unbind();
         super.onPause();
-        mPresenter.unsubscribe();
+    }
+
+    private void bind() {
+        mSubscription = new CompositeSubscription();
```

Comment:
this is correct but you just all those to the view and as i said "V becomes way too complex." thus difficult to main, test and refactor- the very reasons why we try to decouple things.